content/en/users/compute/content-distribution/_index.md

```diff
@@ -46,7 +46,7 @@ Request access to the service sending an email to cvmfs-support@gridpp.rl.ac.uk
 In the email, include the following information:
 
 - Name of the VO or CVMFS repository.
-- Distinguish Name (DN) from your X509 grid certificate.
+- Your `voperson_id` from EGI Check-in.
```

Comment:
i think it's better to add instruction on how to get the voperson_id or point to some existing docs

Reply:
Or maybe in fact it's just easier to replace `voperson_id` by Check-in ID, keeping the same link?

Reply:
yes they are the same:-)

Reply:
OK, I've updated this.